### PR TITLE
Added sign up field validation

### PR DESCRIPTION
**Pull request by ElBmann**

### CODE REVIEW

File: src/components/pages/HomePage.js [hunk 1]
```diff
@@ -11,20 +12,24 @@ export default function HomePage() {
         const data = {
             formName: 'formName', formEmail: email, formMessage: 'New Web3 Signup'
         }
-
-        fetch('https://behcfjv89k.execute-api.us-east-1.amazonaws.com/default/contact', {
+        console.log(process.env)
```
Comment (frankTurtle) [inline src/components/pages/HomePage.js:15]: remove

File: src/components/pages/HomePage.js [hunk 2]
```diff
@@ -11,20 +12,24 @@ export default function HomePage() {
...
+        fetch(HOME_PAGE_FETCH_URL, {
             method: "POST",
             headers: {
                 'Content-Type': 'application/x-www-form-urlencoded',
-                'X-Api-Key': '0vSWfAWdLV38saVPNuShf9tMNzvHfEgs29PMxU9L'
+                'X-Api-Key': HOME_PAGE_API_KEY
             },
             body: JSON.stringify(data)
         }).then(result => {
+            console.log(result)
```
Comment (frankTurtle) [inline src/components/pages/HomePage.js:25]: remove

**Review by frankTurtle — CHANGES_REQUESTED** [review]
Looks good, just remove the `console.log()`

File: src/components/pages/HomePage.js (hunk 2 above)
Comment (ElBmann) [reply src/components/pages/HomePage.js]: Removed

File: src/components/pages/HomePage.js (hunk 1 above)
Comment (frankTurtle) [reply src/components/pages/HomePage.js:15]: @ElBmann what about this one?

**Pull request merged**